scripts/install_deps.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+GLFW_URL="https://github.com/glfw/glfw/archive/refs/tags/3.4.tar.gz"
+GLEW_URL="https://github.com/nigels-com/glew/releases/download/glew-2.2.0/glew-2.2.0.tgz"
+ASSIMP_URL="https://github.com/assimp/assimp/archive/refs/tags/v5.4.0.tar.gz"
```

Comment:
I initially thought that you're gonna do the `git clone`, but release source code should be okay for now 👍

Reply:
git clone is not going to work here, for most libraries maybe this would be ok, by for example for GLEW it is highly recommended to use release snapshots, and if you would try to build master it would fail. Here's related issue: https://github.com/nigels-com/glew/issues/65. In general I think its safer to use snapshot and get stable release than just build branches.